internal/store/postgres/tenant/secret_repository.go

```diff
@@ -126,8 +112,8 @@ func (s SecretRepository) Save(ctx context.Context, tenantSecret *tenant.Secret)
 	}
 
 	insertSecret := `INSERT INTO secret (name, value, type, project_name, namespace_name, created_at, updated_at)
```

Comment:
we have type here still

Reply:
Needs a migration to remove the column. Will add it.